pkg/web/static/js/userProfile.js

```diff
@@ -1,6 +1,8 @@
+import { setSuccessToast } from "./utils.js";
+
 // Helper function to add the same event handler for multiple events
-const addEventListeners = (el, evts, fn) => {
-  evts.split(' ').forEach(e => el.addEventListener(e, fn, false));
+export const addEventListeners = (el, evts, fn) => {
+  evts.split(' ').forEach(e => el?.addEventListener(e, fn, false));
```

Comment:
Optional chaining was added throughout this file to resolve some undefined errors that were appeared in the console.